packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -261,7 +261,7 @@ export interface IContextualMenuItem {
    * Method to custom render this menu item
    * @defaultvalue undefined
    */
-  onRender?: (item: any) => React.ReactNode;
+  onRender?: (item: any, index?: number, hasCheckmarks?: boolean, hasIcons?: boolean) => React.ReactNode;
```

Comment:
@dmarcey how important is this change right now? Can we hold off on it until we can do it as David suggested above?